src/index.css

```diff
@@ -153,7 +153,7 @@ Used to fill vertical space.
 .ui.form .cashflow .cashflow-column-label {
   text-align: left;
   width: auto;
-  padding-left: 1em !important;
+  padding-left: 1em;
```

Comment:
Yay for getting rid of `!important`!! :tada:

Reply:
Agreed. I may add 'index.css' to the code style list. We've got a couple more of those in there from what I could see.